pyanaconda/payload/storage.py

```diff
@@ -0,0 +1,61 @@
+# Storage related code for payload will be contained here.
+#
+# Copyright (C) 2019  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+import os
+import blivet.util
+import blivet.arch
+
+from pyanaconda.anaconda_loggers import get_module_logger
+from pyanaconda.payload.errors import PayloadSetupError
+
+log = get_module_logger(__name__)
+
+
+def get_mount_device(mountpoint):
```

Comment:
Just a note, don't expect that these functions will be provided by the Storage module. It is a very basic functionality that doesn't require a device tree or any additional storage support.

Reply:
We can discuss this later. It is definitely not a priority now.